lndgrpc/client.py

```diff
@@ -16,11 +16,107 @@ def init_wallet(self):
     def generate_seed(self):
         raise NotImplementedError
 
+    #WALLETUNLOCKERRPC
+    @handle_rpc_errors
+    def unlock_wallet(self, **kwargs):
+        # .lnrpc
+        request = walletunlocker.UnlockWalletRequest(
+            **kwargs,
+            # wallet_password=<bytes>,
+            # recovery_window=<int32>,
+            # channel_backups=<ChanBackupSnapshot>,
+            # stateless_init=<bool>,
+        )
+        try:
+            response = self._walletunlocker_stub.UnlockWallet(request)
+            return response
+        except Exception as e:
+            print(e)
+            print("Wallet might already be unlocked")
+
+
+    #WALLETRPC
+    @handle_rpc_errors
+    def next_addr(self, account=""):
+        request = walletkit.AddrRequest(account=account)
+        response = self._walletkit_stub.NextAddr(request)
+        return response
+
+    @handle_rpc_errors
+    def list_accounts(self, **kwargs):
+        request = walletkit.ListAccountsRequest(**kwargs)
+        response = self._walletkit_stub.ListAccounts(request)
+        return response
+
+    #VERRPC
+    @handle_rpc_errors
+    def get_version(self, **kwargs):
+        request = ver.VersionRequest()
+        response = self._version_stub.GetVersion(request)
+        return response   
+
+    # ROUTERRPC
+    @handle_rpc_errors
+    def build_route(self, amt_msat, oid, hop_pubkeys, **kwargs):
+        request = router.BuildRouteRequest(
+            amt_msat=amt_msat,
+            outgoing_chan_id=oid,
+            hop_pubkeys=hop_pubkeys,
+            **kwargs
+        )
+        response = self._router_stub.BuildRoute(request)
+        return response
+
+    @handle_rpc_errors
+    def send_to_route(self,pay_hash,route):
+        request = router.SendToRouteRequest(
+            payment_hash=pay_hash,
+            route=route,
+        )
+        response = self._router_stub.SendToRouteV2(request)
+        return response
+
+    @handle_rpc_errors
+    def send_payment_v2(self,**kwargs):
+        request = router.SendPaymentRequest(**kwargs)
+        last_response = None
+        for response in self._router_stub.SendPaymentV2(request):
+            print(response)
```

Comment:
I think we should not have a print statement here. Maybe instead we use a logger in this file and have this set to debug or info level. That way the caller can limit what they want to see by the log level.